### ISSUE TITLE
.libPaths() warning when restarting R is a little annoying

### ISSUE THREAD

**Issue by CeresBarros** [PredictiveEcology/Require#19]
I've updated R and consequently all my packages and now, every time I (re-)start R this warning is printed:
```
.libPaths() is now: C:/Program Files/R/R-4.0.3/library
```
This is obviously not a _real_ issue, but its a bit annoying... Is it necessary? And what is the goal of having this printed all the time?

Thanks!

**Comment by tati-micheletti** [comment]
I actually think it is good that it is printed every time, especially if you are switching between projects that have different libraries and therefore, different package versions. It gives me the assurance I need that I am not using the wrong package version in each project. But I do get why people would get annoyed, as it prints out every single time you restart a session. :)

**Comment by tati-micheletti** [comment]
Maybe an option to turn it silent would be handy?

**Comment by CeresBarros** [comment]
I think for the "average" R user they don't care about this message... but then maybe they won't install `Require`...

**Comment by achubaty** [comment]
i still think this behaviour should be opt-in rather than the default

**Comment by achubaty** [comment]
It's also not doing I expect it should. It's setting the system libraries as default before the user library, and there is no project library.

I expect the order should be project library (e.g., `packages` or similar) , then the user's personal library (in `~/R/x86_64-pc-linux-gnu-library/4.0`), then system libraries.

**Comment by CeresBarros** [comment]
I don't know if I agree that `Require` should change the default libraries to any other than the system library, by default. That should be something that each user does, if they want to, at the start of their scripts.

**Comment by eliotmcintire** [comment]
FYI: `Require` was never setting a library by itself. The user was doing it with `setLibPaths`. Nevertheless, I have changed the default behaviour to "not be sticky", i.e., the library paths will not persist through a restart of the session. The way `renv` deals with this is that the user runs `init()` which *includes* making the library paths sticky. 

It was always "opt-in", via the user running `setLibPaths`.  The default setting was to make it *sticky*. Now it is not the default.

Re:
> It's also not doing I expect it should. It's setting the system libraries as default before the user library, and there is no project library.

This was a message error, not a functional error. The message has been fixed.
